modules/bim/app/services/bim/ifc_models/update_service.rb

```diff
@@ -43,6 +43,10 @@ def after_perform(service_result)
           # attachments ourselves
           model.attachments.select(&:marked_for_destruction?).each(&:destroy)
 
+          model.conversion_status = ::Bim::IfcModels::IfcModel.conversion_statuses[:pending]
+          model.conversion_error_message = nil
+          model.save
```

Comment:
I'd prefer `model.update(conversion_status: ..., conversion_error_message: ....)` but that's only a nitpick